cmd/notation/blob/inspect_test.go

```diff
@@ -0,0 +1,56 @@
+package blob
```

Comment:
Please add a test where it actually reads signature from a file.

Reply:
Test added